### ISSUE TITLE
Vue.js web icons does not work

### ISSUE THREAD

**Issue by SrdjanMilivojevic** [konpa/devicon#78]
Vue.js web icons does not work when you install it via npm.

**Comment by nabijaczleweli** [comment]
They work for me [here](http://konpa.github.io/devicon/) and, as copied from that page:

```html
<html>
	<head>
		<link rel="stylesheet" href="https://cdn.rawgit.com/konpa/devicon/df6431e323547add1b4cf45992913f15286456d3/devicon.min.css">
	</head>
	<body>
		<i class="devicon-vuejs-plain"></i>
	</body>
</html>
```

**Comment by SrdjanMilivojevic** [comment]
I justs tested that and It works.
The problem seems to be: It doesn't work with NPM !
Mabye the npm repo is not updated at some point...

So, correction: It doesn't work when you install it via npm.

**Comment by SrdjanMilivojevic** [comment]
Problem solved after reading this tread : https://github.com/konpa/devicon/issues/73

**Comment by amacado** [comment]
Closing this issue, since @SrdjanMilivojevic stated it was solved in #73
